docs/management/updates.md

```diff
@@ -30,6 +30,31 @@ If your version is lower than `8.2`, it will not receive updates anymore. To kee
 
 Your dom0 system must either have access to the internet, or to a local mirror. In the second case, make sure to update the `baseurl` values in `/etc/yum.repos.d/xcp-ng.repo` to make them point at the local mirror, and keep the mirror regularly synced.
 
+#### Proxy configuration
+
+If your hosts need to use a proxy to be able to access the repositories, there are multiple ways to configure it depending on your use case:
+
+For all yum repositories:
+- Set `proxy=<proxy_url>` in `[main]` section of `/etc/yum.conf`
+
+For specific repositories:
+- Set `proxy=<proxy_url>` in the repository section of your choice in files present in `/etc/yum.repos.d`
+
+Set a system-wide proxy that will be used for everything, not only yum:
+- Add you can export lines for `http_proxy` and `https_proxy` variables in `/etc/environment`
+```bash
+export http_proxy=<proxy_url>
+export https_proxy=<proxy_url>
+```
+
+:::warning
+There are caveats for each methods:
+- Updating the `yum` package may overwrite `/etc/yum.conf`
+- Updating the `xcp-ng-release` package may overwrite files in `/etc/yum.repos.d/`
+- Updating the `setup` package may overwrite `/etc/environment`
```

Comment:
Have you checked the way they are package, so that we can say for sure whether they will be overwritten or not? The counterpart when they're not overwritten is you don't get the latest changes and have to look for .rpmnew files (whereas overwritten changes are saved to a .rpmsave file).

Reply:
I only checked which package provides them, so no I'm not sure, I'll try to find out, but not all files are always singlely listed in spec files, so I need to understand what's the default behavior.

Reply:
After checking the spec files, it seem there is a `%config(noreplace)` directive for all these, so should I just remove these 3 lines and keep the warning about upgrades I added for your following comment?

Reply:
Yes:
* they won't get evolutions of the file if updated via regular updates (the new packaged files being written as .rpmnew files instead), so that maintenance updates don't overwrite their settings
* on the contrary, when XCP-ng is upgraded to a newer version, they'll have to re-do their changes

Reply:
Fixed and reworded the sentence I about upgrades to make the distinction between updates and upgrades clear. Rebased on master as well.